configurations/stress/c-s-throughput-cmds-i4i.yaml

```diff
@@ -0,0 +1,4 @@
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -mode cql3 native -rate threads=250 -pop seq=1..30000000"
```

Comment:
can you add a few words here ? about what the difference from the regular commands used in `perf-regression.100threads.30M-keys.yaml` ?

Reply:
it has 250 threads, instead of 100

Reply:
The only changes is number of threads. it increase from 100 to 250.